allensdk/brain_observatory/behavior/data_objects/metadata/behavior_metadata/project_code.py

```diff
@@ -0,0 +1,41 @@
+from pynwb import NWBFile
+
+from allensdk.core import DataObject
+from allensdk.core import \
+    LimsReadableInterface
+from allensdk.internal.api import PostgresQueryMixin
+from allensdk.core import NwbReadableInterface
+
+
+class ProjectCode(DataObject, LimsReadableInterface, NwbReadableInterface):
+    """Unique identifier for the project this BehaviorSession is associated
+    with. Project ids can be used internally to extract a project code.
+
+    If project_Code is null/None, we set the value to string 'None'.
```

Comment:
Curious why you don't leave the value as `None`

Reply:
It makes the NWB storage simpler. The extensions never wanted to accept a `None` value and returning a default was the easier solution vs trying to track down it down in all the disparate NWB extension schemas.